curvefs/src/client/dentry_cache_manager.cpp

```diff
@@ -102,19 +137,19 @@ CURVEFS_ERROR DentryCacheManagerImpl::DeleteDentry(
 }
 
 CURVEFS_ERROR DentryCacheManagerImpl::ListDentry(
-    uint64_t parent, std::list<Dentry> *dentryList) {
+    uint64_t parent, std::list<Dentry> *dentryList, uint32_t limit) {
     bool perceed = true;
     MetaStatusCode ret = MetaStatusCode::OK;
     dentryList->clear();
     std::string last = "";
+    limit = (limit == 0) ? maxListDentryCount_ : limit;
```

Comment:
just move maxListDentryCount_ to fuse_client.cpp will better, and you can delete DCacheOption.